cmd/mtnormalise.cpp

```diff
@@ -64,16 +66,25 @@ void usage ()
     + Option ("niter", "set the number of iterations. (default: " + str(DEFAULT_MAIN_ITER_VALUE) + ")")
     + Argument ("number").type_integer()
 
+    + Option ("value", "specify the (positive) reference value to which the summed tissue compartments will be normalised. "
```

Comment:
I find `-value` a bit nondescript. I'd rather use `-reference` or `-sum` or something more expressive?

Reply:
:+1: for `-reference`

Reply:
Happy with '-reference'; definitely *not* '-sum' though: that suggest too much that the output will strictly sum to that value, which `mtnormalise` by definition of its very purpose does not intend to achieve.  The reason for the original name '-value` was discussed among Dave, Rami and myself: to indicate that it's merely a reference value (in a way, this is a sort of "necessary" evil for a normalisation step like this to act across subjects and populations; the reference value is the (not so) hidden means by which they communicate to each other indirectly, yet without requiring a script/command like the old strategy that needs to take all data at once and doesn't allow for future addition of individual subjects without rerunning everything).  So well, long story short: "`-value`" intended to come across as "just reference value".  `-reference` does that as well, so happy with that if that makes things more clear.

Reply:
Looks like we go for `-reference`, then.